### DISCUSSION TITLE
Output Module(defining datetime)

### DISCUSSION THREAD

**Discussion by Dinho4** [pyswmm/pyswmm#363] [Q&A] [answered]
Whenever I try to get my node series using Output module, I get the following:

>>> from swmm.toolkit.shared_enum import NodeAttribute
>>> from pyswmm import Output
>>> with Output('C:/Users/Malesela/Downloads/CIV5000WProject.out') as out:
...     ts = out.node_series('12', NodeAttribute.INVERT_DEPTH, datetime(2018, 8, 25, 0, 9, 59), datetime(2018, 8, 26, 23, 54, 59))
...     for index in ts:
...         print(index, ts[index])
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
NameError: name 'datetime' is not defined

**Comment by dokzai** [comment, marked as answer]
Hey, try importing datetime before executing the script.
`from datetime import datetime`

**Reply by Dinho4** [reply]
Thanks @dokzai, it's all sorted.
